contrib/julia-config.jl

```diff
@@ -0,0 +1,52 @@
+const options =
+[
+   "--cflags",
+   "--ldflags"
+];
+
+function imagePath()
+   opts = Base.compileropts();
+   image_path = bytestring(opts.image_file);
+end
+
+function libDir()
+   @windows_only dir = joinpath(Base.JULIA_HOME, "julia");
```

Comment:
This isn't right - there's no `julia` subdirectory of `JULIA_HOME`, on Windows the executables and shared libraries are all in bin.

Also please use 4-space indents for consistency with the rest of base.